code/game/machinery/requests_console.dm

```diff
@@ -411,6 +518,76 @@ var/list/obj/machinery/requests_console/allConsoles = list()
 			var/obj/item/weapon/stamp/T = O
 			msgStamped = text("<font color='blue'><b>Stamped with the [T.name]</b></font>")
 			updateUsrDialog()
+	if (istype(O, /obj/item/weapon/paper_bundle))
+		if(lid)	//More of that restocking business
+			var/obj/item/weapon/paper_bundle/C = O
+			paperstock += C.amount
+			user.drop_item(C)
+			del(C)

```

Comment:
Oh, quick note. All instances of del(arg) should be swapped for qdel(arg). I should probably mention this to everyone, weee~